packages/devtools_app/test/timeline_streams_test.dart

```diff
@@ -17,6 +18,8 @@ void main() async {
     const FlutterRunConfiguration(withDebugger: true),
   );
 
+  setGlobal(MessageBus, MessageBus());
```

Comment:
this shouldn't be necessary now that the global is set in FlutterTestEnvironment

Reply:
done